source/ui/IDEAS/IDEASXASScanConfigurationView.cpp

```diff
@@ -398,3 +418,8 @@ void IDEASXASScanConfigurationView::onFluorescenceChoiceChanged(int id)
 {
 	configuration_->setFluorescenceDetector(id);
 }
+
+void IDEASXASScanConfigurationView::onDetectorSettingsChange(const QString &args)
```

Comment:
I don't think this slot is connected to any signal. As this is the only means through which the detectorSettings_ label is updated, I believe it will always be blank.